src/api/Game.js

```diff
@@ -0,0 +1,38 @@
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+const getGameId = async (gameName) => {
+  try {
+    const headers = {
+      method: 'POST',
+      body: JSON.stringify({ name: gameName }),
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+    };
+    const response = await fetch(`${url}/games`, headers);
+    const result = await response.json();
+    return result;
+  } catch (error) {
+    return null;
+  }
+};
+
```

Comment:
### Broken Logo :arrow_down:
![image](https://user-images.githubusercontent.com/93039359/216095711-f65cfd8b-d25c-4f60-8555-4d4efca9bfe8.png)

**[Optional]**

- [ ] You have a broken link of your readme logo which is making your readme not look professional :thinking: Kindly remove it or fix it or use your own logo (If you have) :+1: